matrix/all-in-one.yaml

```diff
@@ -0,0 +1,534 @@
+    default: false
+    type: boolean
+    category: diagnostic logging
+  - name: MANUAL SETTINGS
+    description: How to handle https://docs.splunk.com/observability/en/gdi/get-data-in/application/otel-dotnet/configuration/advanced-dotnet-configuration.html#environment-variables-for-manual-installation?
+    default: N/A
+    type: N/A
+    category: manual
+    toDoComment: See description
+instrumentations:
+  - keys:
+      - ASPNET
+    library: ASP.NET Framework (.NET Framework)
+    version: See :ref:`dotnet-otel-versions`
+    comment: MVC / WebApi (Only integrated pipeline mode supported). Metrics requires trace instrumentation.
+    stability: beta
+    supportabilityLevel: community
+    dependencies:
+      - link: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.AspNet
+        version: 1.0.0-rc9.9
+        stability: beta
+      - link: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule
+        version: 1.0.0-rc9.9
+        stability: beta
+    toDoComment: version with ref link
+    signals:
+      - traces: 
+      - metrics: 
+  - keys:
+      - ASPNETCORE
+    library: ASP.NET Core
+    version: See :ref:`dotnet-otel-versions`
+    comment: Metrics automatically activates `Microsoft.AspNetCore.Hosting.HttpRequestIn` spans.
+    stability: beta
+    supportabilityLevel: community
+    dependencies:
+      - link: https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Instrumentation.AspNetCore
+        version: 1.5.1-beta.1
+        stability: beta
+    toDoComment: reflink in version
+    signals:
+      - traces: 
+      - metrics: 
+  - keys:
+      - AZURE
+    library: Azure SDK
+    version: '`Azure.` prefixed packages, released after October 1, 2021'
+    stability: beta
+    supportabilityLevel: third-party
+    signals:
+      - traces: 
+  - keys:
+      - ELASTICSEARCH
+    library: Elastic.Clients.Elasticsearch
+    version: 8.0.0 and higher
+    stability: beta
+    supportabilityLevel: third-party
+    signals:
+      - traces: 
+  - keys:
+      - ENTITYFRAMEWORKCORE
+    library: Microsoft.EntityFrameworkCore
+    version: 6.0.12 and higher
+    comment: Not supported on .NET Framework
+    stability: beta
+    supportabilityLevel: community
+    dependencies:
+      - link: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/tree/main/src/OpenTelemetry.Instrumentation.EntityFrameworkCore
+        version: 1.0.0-beta.7
+        stability: beta
+    signals:
+      - traces: 
+  - keys:
+      - GRAPHQL
+    library: GraphQL
+    version: 7.5.0 and higher
+    comment: Not supported on .NET Framework
+    stability: beta
+    supportabilityLevel: third-party
+    signals:
+      - traces: 
+  - keys:
+      - GRPCNETCLIENT
+    library: Grpc.Net.Client
+    version: 2.52.0 to 3.0.0
```

Comment:
`supported-versions`?

Reply:
Camel Case vs hyphenated will be separated discussion

a11824defcfb162a97906b3fa7992b32d83ef9b0